### PR TITLE
Add 'theme' option and two new themes

### PR DESCRIPTION
**Pull request by lscheibel**
This PR will add the ability to create themes and adds two new ones already. Also adds a background to the standard theme and explain how to create custom themes in `THEMING.md`. 

PS: please check for spelling errors 😬

nord-light:
image

nord-dark:
image
image

### CODE REVIEW

File: CONFIGURATION.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Your configuration overrides the default configuration, which can be found
 gitlabApi: https://gitlab.example.com/api/v4
 
 # Required
+# Can be generated here: https://gitlab.com/profile/personal_access_tokens At least api_read scope is needed
```
Comment (timoschwarzer) [inline CONFIGURATION.md:14]: Please use `https://gitlab.example.com/profile/personal_access_tokens` to make clear that you cannot get access tokens for your own instance on gitlab.com

File: CONFIGURATION.md [hunk 2]
```diff
@@ -169,6 +170,12 @@ projectScopeId: null
 # If projectScope is "groups" you might be interested in using includeSubgroups to include
 # projects from all subgroups within the group specified in projectScopeId.
 includeSubgroups:  false
+
+# Predefined Theme to use, if not set, the default theme will be used.
```
Comment (timoschwarzer) [inline CONFIGURATION.md:174]:
```suggestion
# Predefined theme to use. If null, the default theme will be used.
```

File: CONFIGURATION.md [hunk 3]
```diff
@@ -169,6 +170,12 @@ projectScopeId: null
...
+# Available themes: nord-dark, nord-light
+# Heads-up: Remember that you can always add your custom styles on top
+# but please feel free to submit your own theme as 'src/themes/<your-theme-name>.theme.scss'!
```
Comment (timoschwarzer) [inline CONFIGURATION.md:177]:
```suggestion
# You can override styles in the settings or submit your own theme here: https://git.io/JUOFb
```

File: THEMING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Theming
+
+If you think that you're custom styles would be appreciated but others too, you can submit your own theme as a pull-request!
```
Comment (timoschwarzer) [inline THEMING.md:3]:
```suggestion
You can submit your own themes as a pull request to make it available to all users.
```

File: THEMING.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+## Creating a Theme
+
+To create a theme simply create a `my-theme.theme.scss` file under `src/themes/`
+and add your theme name as a wrapper around everything that you'll add.
```
Comment (timoschwarzer) [inline THEMING.md:8]:
```suggestion
and add a global class selector named after your theme.
```

File: THEMING.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+```scss
+/* src/themes/my-theme.theme.scss */
+
+.my-theme {
+  // Your styles here
+}
+```
+
+To use your theme simply add the name of your theme in the config:
```
Comment (timoschwarzer) [inline THEMING.md:18]:
```suggestion
Enable your theme in your configuration to see your changes:
```

File: THEMING.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+```yaml
+theme: my-theme
+```
+
+## Variables
+
+We are using [css-custom-properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) for variables.
```
Comment (timoschwarzer) [inline THEMING.md:26]:
```suggestion
Themes use [CSS Custom Properties](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) for variables.
```

File: THEMING.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+There are quite a few variables out there to make your life easier.
```
Comment (timoschwarzer) [inline THEMING.md:28]:
```suggestion
There are quite a few variables out there to make theme development easier.
```

File: THEMING.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+To see which ones are available please refer to the existing themes
+or look in the dev-tools.
```
Comment (timoschwarzer) [inline THEMING.md:30]:
```suggestion
Take a look at existing themes for a list of variables or check out your browser's dev tools.
```

File: THEMING.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+Please be very considerate when adding new variables as the trend
+is going to be to have fewer variables than more
+for maintainability purposes.
+
```
Comment (timoschwarzer) [inline THEMING.md:35]:
```suggestion
```

File: THEMING.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+## Maintaining Your Theme
+
+Unfortunately we can't guaranty that your theme is not going to break
+when adding new features or options. But we'd like to encourage everyone
+to create pull-requests to update and fix any issues with any of the themes.
```
Comment (timoschwarzer) [inline THEMING.md:40]:
```suggestion
```

File: src/themes/nord-dark.theme.scss [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+$frost-light-blue: $nord9;
+$frost-dark-blue: $nord10;
+
+$aurora-red: $nord11;
+$aurora-orange: $nord12;
+$aurora-yellow: $nord13;
+$aurora-green: $nord14;
+$aurora-purple: $nord15;
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@700&family=Roboto:ital@0;1&display=swap');
```
Comment (timoschwarzer) [inline src/themes/nord-dark.theme.scss:23]: Please add fonts as a direct dependency

File: src/themes/nord-light.theme.scss [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+$frost-light-blue: $nord9;
+$frost-dark-blue: $nord10;
+
+$aurora-red: $nord11;
+$aurora-orange: $nord12;
+$aurora-yellow: $nord13;
+$aurora-green: $nord14;
+$aurora-purple: $nord15;
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Condensed:wght@700&family=Roboto:ital@0;1&display=swap');
```
Comment (timoschwarzer) [inline src/themes/nord-light.theme.scss:23]: fonts

**Review by timoschwarzer — CHANGES_REQUESTED** [review]

File: THEMING.md (hunk 4 above)
Comment (lscheibel) [reply THEMING.md:3]: Changed 'it' to 'them'

**Comment by lscheibel** [comment]
All done

**Comment by timoschwarzer** [comment]
This needs a rebase

**Comment by lscheibel** [comment]
This PR now also fixes a bug introduced with #156 where clearTimeout was called incorrectly leading to settings being applied incompletely, forcing a reload

**Comment by timoschwarzer** [comment]
Thanks!

**Pull request merged**